src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -465,7 +468,7 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
         /// This signal is emitted when response policy would be decided.<br />
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public event EventHandler<WebViewResponsePolicyDecidedEventArgs> ResponsePolicyDecided
+        public event EventHandler<WebViewPolicyDecidedEventArgs> ResponsePolicyDecided
```

Comment:
could you add **sample code or internal TCT** in order to verify your implementation?

Reply:
Done.